quickstart-book/src/1-intro/application.md

```diff
@@ -7,8 +7,8 @@ The initial goal is to open a window using Bevy!
 Let's start a new project with Bevy
 
 ```sh
-cargo new bevy_workshop-rustweek-2025
-cd bevy_workshop-rustweek-2025
```

Comment:
Should the quickstart crate be called `book`?

Could prob be changed in another phase

Reply:
That's probably better, or maybe just `game` so we don't confuse it with the actual book?